### ISSUE TITLE
No module named 'cufflinks'

### ISSUE THREAD

**Issue by malapradej** [santosjorge/cufflinks#90]
I've installed plotly version 2.4.1 and cufflinks version 2.2.1 through conda. When importing cufflinks in Jupyter using

    import plotly
    import cufflinks as cf

I get 

    ModuleNotFoundError: No module named 'cufflinks'

I'm running Ubuntu 16.04.

**Comment by timkpaine** [comment]
I don't think you have, can you send the output of `pip freeze` if you have pip and also `condo list`

**Comment by santosjorge** [comment]
I dont think there is a conda package for cufflinks. Have you tried with just a pip install?

**Comment by amruizva** [comment]
Have you tried?

conda install -c bioconda cufflinks

**Comment by malapradej** [comment]
@amruizva . It worked, thanks.

**Comment by tijptjik** [comment]
The issue is that in the conda repository, _cufflinks_ refers to the bioconda [package](https://anaconda.org/bioconda/cufflinks), instead of the santosjorge's cufflinks you meant to install. Refer to these instructions to install [cufflinks-py](https://anaconda.org/conda-forge/cufflinks-py) with conda.

**Comment by dmrochon** [comment]
> The issue is that in the conda repository, _cufflinks_ refers to the bioconda [package](https://anaconda.org/bioconda/cufflinks), instead of the santosjorge's cufflinks you meant to install. Refer to these instructions to install [cufflinks-py](https://anaconda.org/conda-forge/cufflinks-py) with conda.

The cufflinks-py link WORKED when everything else failed!! THANK YOU

**Comment by anand-kumar007** [comment]
I just tried random things for installing cufflinks but none of them worked.
so better you install the cufflinks in a separate environment. 

OR
remove all the cufflinks versions if you have any and then 
either install using pip or conda and choose your 
python Kernal accordingly either conda or the normal one !!!

I did the same thing and it worked.

**Comment by ARodenboog** [comment]
I had the same issue, where cufflinks shows up in pip list, but could not be imported. Tried multiple ways of installing cufflinks, via different channels with conda or with pip. 
In the end I made a new conda environment, and installed cufflinks from github using pip from within the jupyter notebook:

`!pip install "git+git://github.com/santosjorge/cufflinks.git#egg=cufflinks" -U`

This did finally did the trick! Hope this helps someone else.
